src/main/java/levy/art/json/RijksCollection.java

```diff
@@ -1,9 +1,12 @@
 package levy.art.json;
 
 import com.google.gson.annotations.SerializedName;
-import levy.art.json.ArtObject;
 
 public class RijksCollection {
-    public ArtObject[] artObjects;
-}
+    private ArtObject[] artObjects;
```

Comment:
Why did you change this from public to private?

Reply:
didnt seem like it needed to be public

Reply:
should it go back to public